packages/logs/lib/models/logContextGetter.ts

```diff
@@ -39,30 +40,19 @@ export const logContextGetter = {
             report(new Error('failed_to_create_operation', { cause: err }), { id: msg.id });
         }
 
-        return new LogContext({ parentId: msg.id, operation: msg }, options);
+        return new LogContextOrigin({ operation: msg }, options);
     },
 
     /**
      * Return a Context without creating an operation
      */
-    async get({ id }: { id: OperationRow['id'] }, options?: Options): Promise<LogContext> {
-        try {
-            if (envs.NANGO_LOGS_ENABLED) {
-                const store = await getKVStore();
-                const indexName = await store.get(`es:operation:${id}:indexName`);
-
-                const operation = await getOperation({ id, indexName });
-                return new LogContext({ parentId: id, operation }, options);
-            }
-        } catch (err) {
-            report(new Error('failed_to_get_operation', { cause: err }), { id });
-        }
-
-        // If it failed, we create a fake operation for now
-        return new LogContext({ parentId: id, operation: { id: nanoid(), createdAt: new Date().toISOString() } as OperationRow }, { ...options, dryRun: true });
+    async get({ id, accountId }: { id: OperationRow['id']; accountId?: number | undefined }, options?: Options): Promise<LogContext> {
+        const split = id.split('_');
+        const createdAt = split[0] ? new Date(parseInt(split[0], 10)).toISOString() : new Date().toISOString(); // Fallback to default date
```

Comment:
I don't know why we didn't think about it earlier 😝

Reply:
yes and no, I didn't think about it at the time because I knew I could remove the ID and get better performance at indexing. So in my mind, it was about to get removed. However, the simplicity probably far outweighs the perf gain :D Will see 🤞🏻